layouts/partials/head/favicons.html

```diff
@@ -1,5 +1,6 @@
 <meta name="theme-color" content="{{ $.Site.Params.themeColor }}">
-<link rel="apple-touch-icon" sizes="180x180" href="{{ "apple-touch-icon.png" | relURL }}">
-<link rel="icon" type="image/png" sizes="32x32" href="{{ "favicon-32x32.png" | relURL }}">
-<link rel="icon" type="image/png" sizes="16x16" href="{{ "favicon-16x16.png" | relURL }}">
+<!--link rel="apple-touch-icon" sizes="180x180" href="{{ "apple-touch-icon.png" | relURL }}"-->
```

Comment:
If this isn't used then delete it, don't just comment it out.

Reply:
Done